crates/evm/evm/src/executors/fuzz/mod.rs

```diff
@@ -193,17 +205,23 @@ impl FuzzedExecutor {
             }
             Err(TestError::Fail(reason, _)) => {
                 let reason = reason.to_string();
-                result.reason = (!reason.is_empty()).then_some(reason);
+                result.reason = (!reason.is_empty()).then_some(reason.clone());
 
-                let args = if let Some(data) = calldata.get(4..) {
-                    func.abi_decode_input(data, false).unwrap_or_default()
+                if reason == "Timeout reached" {
```

Comment:
Not sure if this is janky? It does seem to work though.

Reply:
agree should be good, maybe we could have a constant to use on both places where we fail test and where we check the result